client/services/FeedbackGeneratorService.js

```diff
@@ -222,6 +225,43 @@ tie.factory('FeedbackGeneratorService', [
         feedback.setSyntaxErrorIndex(feedback.getParagraphs().length - 1);
         return feedback;
       },
+      getPrereqFailureFeedback: function(codePrereqCheckResult) {
+        var feedback = FeedbackObjectFactory.create(false);
+        var preReqFailures = codePrereqCheckResult.getPrereqCheckFailures();
+
+        /* This function shouldn't be invoked unless there is at least one
```

Comment:
I don't understand. Wouldn't it make more sense to throw an Error in this case if we require the input arg to have at least one element?